app/views/trip_families/_people_to_allocate.html.erb

```diff
@@ -0,0 +1,7 @@
+<tr>
+  <td><%= @trip_family.to_allocate(params[:trip_id])["girls"]%></td>
+  <td><%= @trip_family.to_allocate(params[:trip_id])["boys"]%></td>
+  <td><%= @trip_family.to_allocate(params[:trip_id])["teachers"]%></td>
+  <td><%= @trip_family.to_allocate(params[:trip_id])["bus_drivers"]%></td>
+  <td><%= @trip.school_name %></td>
+</tr>
```

Comment:
This partial is not currently used and should be removed.

Reply:
Done in sha `e2a4c4e`